lib/matplotlib/text.py

```diff
@@ -1103,7 +1103,7 @@ def set_math_fontfamily(self, fontfamily):
             The name of the font family.
 
             Available font families are defined in the
-            :ref:`matplotlibrc.template file
+            :ref:`lib/matplotlib/mpl-data/matplotlibrc file
```

Comment:
Similar to the font_manager.py change, I think including the Github path here would just confuse the user.
https://output.circle-artifacts.com/output/job/be496fb6-3b21-4e89-83df-1e83d5755dcd/artifacts/0/doc/build/html/api/text_api.html#matplotlib.text.Text.set_math_fontfamily